src/main/java/CliDisplay.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Scanner;
+
+public class CliDisplay implements Display {
+
+    public void sendMessageToDisplay(String message) {
+       System.out.print(message);
```

Comment:
You have System.out.print here and in the method below. If you have this method then why not use it below?